src/test/article.spec.js

```diff
@@ -0,0 +1,59 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import app from '../index';
+import BaseRepository from '../repository/base.repository';
+import { createUser, createArticle, generateArticle } from './utils/helpers';
+import db from '../database/models';
+import helper from '../helpers/utils';
+
+const ARTICLES_API = '/api/v1/articles';
+
+chai.use(chaiHttp);
+
+const server = () => chai.request(app);
+
+describe('GET api/v1/articles', () => {
+  beforeEach(async () => {
+    await db.Article.destroy({ cascade: true, truncate: true });
+    await db.User.destroy({ cascade: true, truncate: true });
+  });
+
+  it('should get a list of all articles', async () => {
```

Comment:
Good job, I like this test!

Reply:
good job I see you have implemented what TTL ask you to implement.